src/cmd/run.js

```diff
@@ -71,24 +74,17 @@ export async function run (componentPath, args) {
         throw e;
       }
     `);
-    await chmod(runPath, 0o777);
 
-    cp = spawn(argv0, [runPath, ...args], { stdio: 'inherit' });
+    process.exitCode = await new Promise((resolve, reject) => {
+      const cp = fork(runPath, args);
```

Comment:
We still need to set `stdio: 'inherit'`.

Reply:
The [Node.js docs](https://nodejs.org/api/child_process.html#child_processforkmodulepath-args-options) say that `'inherit'` is the default for `fork()`.

Reply:
Even though this seems unnecessary (and, if it was, the test should have failed), I made a new commit to explicitly set this value.